pytypes/typechecker.pyi

```diff
@@ -0,0 +1,48 @@
+# Generated using mypy stubgen, then editing to add types for some definitions.
+
+import __builtin__ as builtins
+from . import type_util, util
+from .exceptions import InputTypeError, OverrideError, ReturnTypeError
+from .type_util import TypeAgent, deep_type, get_orig_class, has_type_hints, is_Tuple, is_builtin_type, type_str
+from .util import getargspecs
+from types import ModuleType
+from typing import Any, Optional, Callable, TypeVar, Union
+from warnings import warn
+
+_python___import__ = builtins.__import__
+
+# Placeholders typed as Any just declare they exist to avoid "no such attribute"
+# errors from attempts to reference names missing from pyi.
+# TODO: Fill in proper types.
```

Comment:
I rather thought of a on-liner to be used as a marker for each not-finally-correctly-typed function or other member. However now it seems that ``_pytypes___import__`` and stuff in ``_DelayedCheck`` are the only items not yet properly typed. Make it clear that the comment applies to these and *only* to these.

Reply:
Okay, I mentioned those two specifically in TODO comments and resolved the rest. (IIUC the remaining `Any` types besides those actually can accept any type.)